File: WOT-iOS/WOTData/WOTData/model/subclasses/ListSetting+CoreDataProperties.m
//
//  ListSetting+CoreDataProperties.m
//  WOTData
//
//  Created on 8/28/18.
//  Copyright © 2018. All rights reserved.
//
//

#import "ListSetting+CoreDataProperties.h"

@implementation ListSetting (CoreDataProperties)

+ (NSFetchRequest<ListSetting *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"ListSetting"];
}

@dynamic ascending;
@dynamic key;
@dynamic orderBy;
@dynamic type;
@dynamic values;

@end

File: WOT-iOS/WOTApi/source/response/managedObjects/autogenerated/ListSetting+CoreDataProperties.swift
//
//  ListSetting+CoreDataProperties.swift
//  WOTData
//
//  Created by Pavel Yeshchyk on 5/1/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

import CoreData

public extension ListSetting {
    @nonobjc class func fetchRequest() -> NSFetchRequest<ListSetting> {
        return NSFetchRequest<ListSetting>(entityName: "ListSetting")
    }

    @NSManaged var ascending: NSNumber?
    @NSManaged var key: String?
    @NSManaged var orderBy: NSNumber?
    @NSManaged var type: String?
    @NSManaged var values: String?
}
